ideascube/blog/forms.py

```diff
@@ -15,6 +15,7 @@ class Meta:
             # control over the format to bypass L10N.
             "published_at": forms.DateInput(format='%Y-%m-%d'),
             "lang": LangSelect,
+            "summary": RichTextEntry,
```

Comment:
This thing about sometimes passing the class and sometimes passing an instance of the class is bugging me. 

But I guess I need to just get on with Django... 😞